NEWS.md

```diff
@@ -1,5 +1,20 @@
 # EpiNow2 (development version)
 
+## Model changes
+
+- The models now supports more complex accumulation patterns, as well as mixtures of accumulation and missingness via the new `fill_missing()` function and a logical `accumulate` column that can be supplied with the data. By @sbfnk in # and reviewed by @.
```

Comment:
I agree but we do need to be careful as we don't support some aggregation schemes that are not accumulation (i.e non-consecutive aggregation)

Reply:
Sounds like a good idea to mention here if we think it's something people search for but in other places we should decide on one term (and make sure we explain it well / better) as otherwise people might just be confused about whether they mean the same or not.

Reply:
```suggestion
- The models now supports more complex patterns of aggregating reported cases by accumulating them over multiple time points, as well as mixtures of accumulation and missingness via the new `fill_missing()` function and a logical `accumulate` column that can be supplied with the data. By @sbfnk in #851 and reviewed by @seabbs and @jamesmbaazam..
```